### PR TITLE
Detekt CI rules

### PR DESCRIPTION
**Pull request by Vacxe**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,5 +1,16 @@
 version: 2
 jobs:
+  code style check:
```
Comment (Unlimity) [inline .circleci/config.yml]: are spaces allowed in the name of the job?

File: build.gradle [hunk 2]
```diff
@@ -4,6 +4,9 @@ buildscript {
     repositories {
         jcenter()
         google()
+        maven {
+            url "https://plugins.gradle.org/m2/"
```
Comment (Unlimity) [inline build.gradle]: Could you please format it to the single line?

File: .circleci/config.yml (hunk 1 above)
Comment (Vacxe) [reply .circleci/config.yml]: Look like allowed. Just for improve readability in CI report here

**Comment by Vacxe** [comment]
See: #144

**Pull request closed without merging**